services/app/src/static/style/header.css

```diff
@@ -0,0 +1,54 @@
+#header {
+  width: 100%;
+  height: 80px;
+  top: 0%;
+  left: 0%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #f3f6fa;
+  position: absolute;
+  flex-grow: 1;
+  min-width: 800px;
+}
+#logo {
+  margin-left: 64px;
+}
+#seperator {
+  margin-left: 16.5px;
+  margin-right: 16.5px;
+}
+#roomName {
+  margin-right: 18px;
+}
+#copyLink {
+  margin-right: 20.28px;
+}
+#settings {
+  margin-right: 20.28px;
+}
+#letter {
```

Comment:
`avatar` sounds like more appropriate name here. 

I would prefer css `class` instead of `id`, as we probably will have multiple `avatar`s displayed on the screen.